modules/task/README.md

```diff
@@ -37,13 +39,13 @@ func newBackend(host service.Host) (task.Backend, error) {
   return b, nil
 }
 
-func runActivity(input string) error {
+func runActivity(ctx context.Context, input string) error {
   // do things and calculate results
   results := "results"
-  return task.Enqueue(updateCache, input, results)
+  return task.Enqueue(updateCache, ctx, input, results)
```

Comment:
I prefer second here because I want the enqueue arguments to map to the actual function. It's more intuitive to see
task.Enqueue(fn, arg1 arg2)
with
func fn(arg1 context.Context, arg2 string)